README.md

```diff
@@ -127,6 +128,19 @@ Are you sure? y
 Assuming MySQL is running locally, the following command runs all of the unit
 tests for the code, and should complete successfully:
 
+### Presubmit Checks
+
+Before checking in code, please ensure that the presubmit checks pass. 
+
+```
+go get -u github.com/golang/lint/golint
+go get -u github.com/kisielk/errcheck
+go get -u github.com/fzipp/gocyclo
+go get -u github.com/gordonklaus/ineffassign
+go get -u github.com/client9/misspell/cmd/misspell
+./presubmit.sh
+```
+
 ```console
```

Comment:
I think @codingllama is referring to the Markdown language identifier, but I *think* `console` is OK for chunks of interactive stuff (ie. with a shell prompt).